### ISSUE TITLE
Failed to setup:

### ISSUE THREAD

**Issue by jeremeyi** [alandtse/alexa_media_player#2524]
**IMPORTANT: Please search the issues, including closed issues, and the [FAQ](https://github.com/alandtse/alexa_media_player/wiki/FAQ) before opening a new issue. The template is mandatory; failure to use it will result in issue closure.**

**Describe the bug**

On rebooting HA I get "Failed to setup:"  on the devices and services page.  Usually a reload of the integration from there will clear it up.  I uninstalled and reinstalled tonight to see if the issue would be resolved but it is still there.  In my logs are:
Logger: homeassistant.config_entries
Source: config_entries.py:604
First occurred: 12:07:54 AM (1 occurrences)
Last logged: 12:07:54 AM

Error setting up entry ****@yahoo.com - amazon.com for alexa_media
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/alexa_media/__init__.py", line 361, in async_setup_entry
    await login.login(cookies=await login.load_cookie())
                              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/alexapy/alexalogin.py", line 328, in load_cookie
    cookies = pickle.loads(await myfile.read())
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/http/cookies.py", line 312, in __setitem__
    raise CookieError("Invalid attribute %r" % (K,))
http.cookies.CookieError: Invalid attribute 'partitioned'

**To Reproduce**

**Expected behavior**

**Screenshots**

**System details**

- Home Assistant version: 2024.9.1
- alexa_media version: 4.12.12

- alexapy version (from `pip show alexapy` in homeasssistant container or HA startup log):
- Is Amazon [2FA/2SV enabled](https://github.com/alandtse/alexa_media_player/wiki/Configuration#enable-two-factor-authentication-for-your-amazon-account)  (y/n): y
- Amazon Domain:

**Debug Logs (alexa_media & alexapy)**
Please provide [logs](https://github.com/alandtse/alexa_media_player/wiki/FAQ#how-do-i-enable-debug-logging-for-the-component).

**Additional context**

**Comment by danielbrunt57** [comment]
> File "/usr/local/lib/python3.12/http/cookies.py", line 312, in setitem
raise CookieError("Invalid attribute %r" % (K,))
http.cookies.CookieError: Invalid attribute 'partitioned'

This is a 3 month old issue which has been resolved with `alexapy` v1.29.2
Edit the `requirements` in `config/custom_components/alexa_media/manifest.json`:
```
  "requirements": ["alexapy==1.29.2", "packaging>=20.3", "wrapt>=1.14.0"],
```

**Comment by jeremeyi** [comment]
Thanks, that worked!  Sorry for reposting, didn't see it when I searched.
